lib/linter-glsl.js

```diff
@@ -69,13 +69,15 @@ const parseGlslValidatorResponse = (inputs, output) => new Promise((resolve) =>
           const colEnd = colStart;
 
           toReturn.push({
-            type: match[1],
-            text: match[4].trim(),
-            filePath: shader.fullFilename,
-            range: [
-              [lineStart > 0 ? lineStart - 1 : 0, colStart > 0 ? colStart - 1 : 0],
-              [lineEnd > 0 ? lineEnd - 1 : 0, colEnd > 0 ? colEnd - 1 : 0],
-            ],
+            severity: match[1].toLowerCase(),
```

Comment:
We don't have a guarantee that this will be one of the allowed values, this will need some sort of validation here.

Reply:
(Note: Even if `glsl` does _now_, they have been pretty bad about making major changes with no warning before.)